module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -91,92 +93,92 @@ namespace module::behaviour::strategy {
         on<Trigger<VisionBalls>>().then([this](const VisionBalls& balls) {
             if (!balls.balls.empty()) {
-                ballLastMeasured = NUClear::clock::now();
+                ball_last_measured = NUClear::clock::now();
             }
         });
 
         on<Trigger<VisionGoals>>().then([this](const VisionGoals& goals) {
             if (!goals.goals.empty()) {
-                goalLastMeasured = NUClear::clock::now();
+                goal_last_measured = NUClear::clock::now();
             }
         });
 
         // TODO(BehaviourTeam): remove this horrible code
         // Check to see if we are currently in the process of getting up.
-        on<Trigger<ExecuteGetup>>().then([this] { isGettingUp = true; });
+        on<Trigger<ExecuteGetup>>().then([this] { is_getting_up = true; });
 
         // Check to see if we have finished getting up.
-        on<Trigger<KillGetup>>().then([this] { isGettingUp = false; });
+        on<Trigger<KillGetup>>().then([this] { is_getting_up = false; });
 
-        on<Trigger<Penalisation>>().then([this](const Penalisation& selfPenalisation) {
-            if (selfPenalisation.context == GameEvents::Context::SELF) {
+        on<Trigger<Penalisation>>().then([this](const Penalisation& self_penalisation) {
+            if (self_penalisation.context == GameEvents::Context::SELF) {
                 emit(std::make_unique<ResetWebotsServos>());
-                selfPenalised = true;
+                self_penalised = true;
             }
         });
 
-        on<Trigger<Unpenalisation>>().then([this](const Unpenalisation& selfPenalisation) {
-            if (selfPenalisation.context == GameEvents::Context::SELF) {
-                selfPenalised = false;
+        on<Trigger<Unpenalisation>>().then([this](const Unpenalisation& self_penalisation) {
+            if (self_penalisation.context == GameEvents::Context::SELF) {
+                self_penalised = false;
 
                 // TODO(BehaviourTeam): isSideChecking = true;
                 // TODO(BehaviourTeam): only do this once put down
-                unpenalisedLocalisationReset();
+                unpenalised_localisation_reset();
             }
         });
 
 
         on<Trigger<ButtonMiddleDown>, Single>().then([this] {
             log("Middle button pressed!");
-            if (!cfg_.forcePlaying) {
+            if (!cfg.force_playing) {
                 log("Force playing started.");
                 emit(std::make_unique<Nod>(true));
-                cfg_.forcePlaying = true;
+                cfg.force_playing = true;
             }
         });
 
         on<Trigger<KickOffTeam>>().then(
-            [this](const KickOffTeam& kickOffTeam) { team_kicking_off = kickOffTeam.context; });
+            [this](const KickOffTeam& kick_off_team) { team_kicking_off = kick_off_team.context; });
 
         // ********************* Main Loop ********************
         on<Every<30, Per<std::chrono::seconds>>,
```

Comment:
Why every 30 seconds? Is this a NUbots standard, or just a value that we've chosen?

If the latter, it may be worth moving this into a value in the relevant yaml rather than stating 30 here?

Reply:
Its 30 times per second. I think its just arbitrarily chosen and its synced with SimpleWalkPathPlanner

Reply:
I also agree this should be a config value